UI/src/components/molecules/RipaStopDate.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="ripa-stop-date tw-p-4">
+    <ripa-form-header
+      title="Date of Stop"
+      required
+      subtitle="§999.226(a)(10)">
+    </ripa-form-header>
+
+    <v-checkbox
```

Comment:
Should replace this with RipaCheckbox. Props needs to be = for cbLabel and cbValue.